src/Swarm/Language/LSP/Hover.hs

```diff
@@ -101,10 +124,9 @@ data DocLine a = DocLine
 renderDoc :: DocLine Text -> Text
 renderDoc (DocLine d t)
   | d == 0 = t
-  | d == 1 = "* " <> t
-  | otherwise = indent (d - 1) <> "* " <> t
+  | otherwise = T.drop 2 . indent (max 0 (4 * (d - 1)) + 2) $ "* " <> t
```

Comment:
Is the `max 0` necessary?  If `d > 0` then `4 * (d - 1) >= 0`.

Reply:
Also, I don't understand why we're adding 2 to the indentation and then doing `T.drop 2`.  Don't those cancel each other out?

Reply:
Huh, I suppose the maximum is redundant.

The indent and drop cancel only for first line of text.
The later lines will be indented two spaces extra.

Reply:
Really? Isn't `T.drop 2 . indent (n + 2)` equivalent to `indent n`?

Reply:
No:
```Haskell
>>> T.putStrLn $ indent 4 "* abc\ndef"
    * abc
    def
>>> T.putStrLn $ T.drop 2 $ indent (4 + 2) "* abc\ndef"
    * abc
      def
```